packages/thirdweb/src/wallets/in-app/web/lib/iframe-wallet.ts

```diff
@@ -242,13 +243,22 @@ export class IFrameWallet implements IWebWallet {
           chain: getCachedChain(tx.chainId),
         });
         const signedTx = await _signTransaction(tx);
+
         const transactionHash = await eth_sendRawTransaction(
           rpcRequest,
           signedTx,
         );
-        return {
+
+        trackTransaction({
+          client,
+          walletAddress: address,
+          walletType: "inApp",
```

Comment:
this could be ecosystem too no?

Reply:
It can't since ecosystem wallets are all on enclave now

Reply:
Right